wcfsetup/install/files/lib/system/html/output/node/HtmlOutputNodeProcessor.class.php

```diff
@@ -289,4 +289,61 @@ protected function invokeHtmlNode(IHtmlNode $htmlNode)
 
         parent::invokeHtmlNode($htmlNode);
     }
+
+    protected function removeTextFormatting(): void
+    {
+        if (!\FORMATTING_REMOVE_COLOR && !\FORMATTING_REMOVE_FONT && !\FORMATTING_REMOVE_SIZE) {
+            return;
+        }
+
+        /** @var list<\DOMElement> */
+        $elementsWithStyles = [];
+        foreach ($this->getXPath()->query("//*[@style]") as $element) {
+            \assert($element instanceof \DOMElement);
+            $elementsWithStyles[] = $element;
+        }
+
+        if ($elementsWithStyles === []) {
+            return;
+        }
+
+        foreach ($elementsWithStyles as $element) {
+            $style = $element->getAttribute("style");
+            $values = \array_map(
+                fn (string $value) => StringUtil::trim($value),
+                \explode(";", $style)
+            );
+
+            $values = \array_filter($values, static function (string $value) {
+                list($keyword,) = \explode(":", $value, 2);
+
+                switch (StringUtil::trim($keyword)) {
+                    case "color":
+                        return \FORMATTING_REMOVE_COLOR === 0;
+
+                    case "font-family":
+                        return \FORMATTING_REMOVE_FONT === 0;
+
+                    case "font-size":
+                        return \FORMATTING_REMOVE_SIZE === 0;
+                }
+
+                return true;
+            });
+
+            $style = \implode(";", $values);
+            if ($style !== "") {
+                $element->setAttribute("style", $style);
+                continue;
+            }
```

Comment:
Is an `else` missing here to remove the attribute if it's empty after processing?

Reply:
Fixed in 0ed94adac902bb751fa7168f5367a6a88eb13d5a